vip-jetpack/vip-jetpack.php

```diff
@@ -218,3 +218,20 @@ function wpcom_vip_disable_jetpack_email_no_recaptcha( $is_enabled ) {
 	return defined( 'RECAPTCHA_PUBLIC_KEY' ) && defined( 'RECAPTCHA_PRIVATE_KEY' );
 }
 add_filter( 'sharing_services_email', 'wpcom_vip_disable_jetpack_email_no_recaptcha', PHP_INT_MAX );
+
+/**
+ * Enable the new Full Sync method on sites with the VIP_JETPACK_FULL_SYNC_IMMEDIATELY constant
+ */
+add_filter( 'jetpack_sync_modules', function( $modules ) {
+	if ( ! defined( 'VIP_JETPACK_FULL_SYNC_IMMEDIATELY' ) || ! VIP_JETPACK_FULL_SYNC_IMMEDIATELY ) {
```

Comment:
Minor: can we reverse this and move the loop inside? I find the multiple `!` checks can be hard to read and somewhat fragile.

```suggestion
	if ( defined( 'VIP_JETPACK_FULL_SYNC_IMMEDIATELY' ) && true === VIP_JETPACK_FULL_SYNC_IMMEDIATELY ) {
```

Reply:
Good call, done in 9c4f8034